website/management/commands/base_lab_seeder.py

```
@@ -0,0 +1,55 @@
+from django.core.management.base import BaseCommand, CommandError
+from website.models import Labs, TaskContent, Tasks
+
+class LabSeederCommand(BaseCommand):
+    lab_name: ClassVar[str | None] = None  # Override in subclass
+    tasks_data: ClassVar[list] = []  # Override in subclass
+
+    def handle(self, *args, **kwargs):
+        try:
+            lab = Labs.objects.get(name=self.lab_name)
+        except Labs.DoesNotExist as err:
+            raise CommandError(f"{self.lab_name} lab not found. Please run create_initial_labs first.") from err
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)